generators/entity-server/templates/quarkus/src/test/java/package/_partials/create.ejs

```diff
@@ -18,20 +18,8 @@
 -%>
     @Test
     public void create<%= entityClass %>() {
```

Comment:
Please keep the validation tests (API should failed when creating with an existing ID)